### PR TITLE
WIP Update dev procedure for libsnowflakeclient

### PR DESCRIPTION
**Pull request by sfc-gh-zli**

### CODE REVIEW

File: ci/dev/README.rst [hunk 1]
```diff
@@ -87,6 +88,7 @@ Setup
     .. code-block:: bash
 
         cd /home/debugger/libsnowflakeclient
+        export BUILD_TYPE='Debug' # or 'Release' for release build
```
Comment (sfc-gh-kbhatia) [inline ci/dev/README.rst:91]: Why this change ?

Comment (sfc-gh-kbhatia) [reply ci/dev/README.rst:91]: Never mind, I realized that this is in the docs.

**Review by sfc-gh-vreddy — CHANGES_REQUESTED** [review]
Secretdetector changes are must for ODBC and other client drivers.
This should NOT be merged.

**Comment by sfc-gh-zli** [comment]
Yes I agree. I used this branch to build the current .a file for XP and I don't intend to merge it. Let me close this PR.

**Comment by sfc-gh-zli** [comment]
But please don't delete this branch yet. This might be useful before we update our workflow of checking in the .a file.

**Pull request closed without merging**